app/localization/translated/be.json

```diff
@@ -78,6 +78,7 @@
   "Common.cancel": "Адмяніць",
   "Common.closeModalWarning": "Вы павінны захаваць змены або адмяніць іх перад закрыццём вокны",
   "Common.confirm": "Пацвердзіць",
+  "Common.dateTime.seconds": "seconds",
```

Comment:
Please add translation from https://github.com/reportportal/service-ui/blob/localization_by/src/main/resources/public/js/src/localizations/be-BY.js#L1997